AutomationTests/Malafi.Tests/Features/HomePage.feature

```diff
@@ -0,0 +1,16 @@
+﻿Feature: HomePage
+
+Click on the Malaf link
+
+Background: Login to Internal Portal and click on Malafi Link
+Given Entered 'blue2'as a username
+	And Enterd 'Ksmc@1234' as password
+	When I cilck on login button
+	And Click on Malafi link
```

Comment:
Please move this line from here to the below scenario. Add When step to the below scenario